psydac/fem/tests/test_spline_interpolation.py

```diff
@@ -79,8 +79,8 @@ def test_SplineInterpolation1D_cosine( ncells, degree, periodic ):
 
 #===============================================================================
 @pytest.mark.parallel
-@pytest.mark.parametrize( "nc1", [5,10,23] )
-@pytest.mark.parametrize( "nc2", [5,10,23] )
+@pytest.mark.parametrize( "nc1", [16,18,23] )
+@pytest.mark.parametrize( "nc2", [16,18,23] )
```

Comment:
We need this after adding this requirement  mentioned in the PR description  `Increase the minimum block size for the cart decomposition to be p+1 in each direction instead of p` otherwise it won't work in parallel

Reply:
You are right I have reduced them [056da00](https://github.com/pyccel/psydac/pull/213/commits/056da00b588a9329c08805700031e27d71058b41)